### PR TITLE
Create action-install-from-pypi.yml

### PR DESCRIPTION
**Pull request by valeriupredoi**
Before you start, please read our [contribution guidelines](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/contributing.html).

**Tasks**

-   [x] [Create an issue](https://github.com/ESMValGroup/ESMValCore/issues) to discuss what you are going to do, if you haven't done so already (and add the link at the bottom)
-   [ ] This pull request has a descriptive title that can be used in a changelog
-   [ ] Add unit tests
-   [ ] Public functions should have a numpy-style docstring so they appear properly in the [API documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.html). For all other functions a one line docstring is sufficient.
-   [ ] If writing a new/modified preprocessor function, please update the [documentation](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/preprocessor.html)
-   [ ] Circle/CI tests pass. Status can be seen below your pull request. If the tests are failing, click the link to find out why.
-   [ ] Codacy code quality checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-   [ ] Please use `yamllint` to check that your YAML files do not contain mistakes
-   [ ] If you make backward incompatible changes to the recipe format, make a new pull request in the [ESMValTool repository](https://github.com/ESMValGroup/ESMValTool) and add the link below

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Related to #731

### CODE REVIEW

**Comment by valeriupredoi** [comment]
hey guys can you pls allow me to merge this - I want to test the cron scheduler for Github Actions and it appears I can test only if I am on master (yeh GA still have limitations) :beer:

File: .github/workflows/action-install-from-pypi.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ on:
   push:
     branches:
     - master
-    - github-actions2
+    - valeriupredoi-patch-1-1
```
Comment (nielsdrost) [inline .github/workflows/action-install-from-pypi.yml:25]: Guess we can remove this, right?

**Comment by valeriupredoi** [comment]
ok merged this one, now we're playing the waiting game see if `master` runs the GA test at midnight :grin:

File: .github/workflows/action-install-from-pypi.yml (hunk 1 above)
Comment (nielsdrost) [reply .github/workflows/action-install-from-pypi.yml]: Thanks!

**Pull request merged**